### PR TITLE
Scroll to bottom button

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### Description

Scroll to bottom button

### Checklist

- [X] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [X] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [X] Comparison screenshots added for visual changes
- [X] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonViewStyle.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package io.getstream.chat.android.ui.messages.view
+
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import androidx.annotation.ColorInt
+import androidx.annotation.StyleableRes
+
+public class ScrollButtonViewStyle(
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonViewStyle.kt:8]: `internal` constructors for these style classes?

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonView.kt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        if (unreadBadgeEnabled) {
+            setUnreadCountValue(unreadCount)
+            setUnreadCountTextViewVisible(unreadCount > 0)
+        } else {
+            setUnreadCountTextViewVisible(false)
+        }
+    }
+
+    private fun setUnreadCountValue(unreadCount: Int) {
+        if (this.unreadCount != unreadCount) {
```
[the remark this answers is not in the document]
Comment (bychkovdmitry) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonView.kt:67]: Just an optimization

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonViewStyle.kt (hunk 1 above)
Comment (zsmb13) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonViewStyle.kt:8]: Sorry, on second thought, can't they be completely `internal`? Do we expose them anywhere?

**Review by zsmb13 — APPROVED** [review]
Nice work! 👍

**Pull request merged**